verify/verify.py

```diff
@@ -218,36 +241,46 @@ async def verify_status(self, ctx: commands.Context):
         role_id = await self.settings.guild(ctx.guild).role()
         if role_id is not None:
```

Comment:
```suggestion
        if role_id:
```
this is more pythonic and much more readable imo, especially as similar logic is applied to all the if statements below